packages/ui-components/src/components/Inputs/AsyncAutocomplete.tsx

```diff
@@ -3,17 +3,19 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
 import throttle from 'lodash.throttle';
```

Comment:
this might be another one that only happens for me but the linter is telling me to install @types/lodash.throttle